http-filter-example/README.md

```diff
@@ -0,0 +1,39 @@
+# Envoy filter example
+
+This project demonstrates the linking of additional HTTP filters with the Envoy binary.
+A new filter `sample` which adds a HTTP header is introduced.
+Integration tests demonstrating the filter's end-to-end behavior are
+also provided.
+
+## Building
+
+To build the Envoy static binary:
+
+1. `git submodule update --init`
+2. `bazel build //http-filter-example:envoy`
+
+## Testing
+
+To run the `sample` integration test:
+
+`bazel test //http-filter-example:http_filter_integration_test`
+
+## How it works
+
+See the [network filter example](../README.md#how-it-works).
+
+## How to write an HTTP filter
+
+- The main task is to write a class 
+ that implements the interface `Envoy::Http::StreamDecoderFilter`,
```

Comment:
Probably. Just say they need to implement the appropriate one in this set of interfaces, and add links to them in the GH repo.